fhir-bulkimportexport/src/main/java/com/ibm/watson/health/fhir/bulkexport/BulkExportBatchLet.java

```diff
@@ -0,0 +1,343 @@
+/**
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.watson.health.fhir.bulkexport;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import javax.batch.api.BatchProperty;
+import javax.batch.api.Batchlet;
+import javax.inject.Inject;
+
+import com.ibm.cloud.objectstorage.ClientConfiguration;
+import com.ibm.cloud.objectstorage.SDKGlobalConfiguration;
+import com.ibm.cloud.objectstorage.auth.AWSCredentials;
+import com.ibm.cloud.objectstorage.auth.AWSStaticCredentialsProvider;
+import com.ibm.cloud.objectstorage.auth.BasicAWSCredentials;
+import com.ibm.cloud.objectstorage.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.ibm.cloud.objectstorage.oauth.BasicIBMOAuthCredentials;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3ClientBuilder;
+import com.ibm.cloud.objectstorage.services.s3.model.Bucket;
+import com.ibm.cloud.objectstorage.services.s3.model.CreateBucketRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.ObjectMetadata;
+import com.ibm.cloud.objectstorage.services.s3.model.PutObjectRequest;
+import com.ibm.watson.health.fhir.config.FHIRConfiguration;
+import com.ibm.watson.health.fhir.config.FHIRRequestContext;
+import com.ibm.watson.health.fhir.model.resource.Resource;
+import com.ibm.watson.health.fhir.model.util.ModelSupport;
+import com.ibm.watson.health.fhir.persistence.FHIRPersistence;
+import com.ibm.watson.health.fhir.persistence.context.FHIRPersistenceContext;
+import com.ibm.watson.health.fhir.persistence.context.FHIRPersistenceContextFactory;
+import com.ibm.watson.health.fhir.persistence.helper.FHIRPersistenceHelper;
+import com.ibm.watson.health.fhir.persistence.helper.FHIRTransactionHelper;
+import com.ibm.watson.health.fhir.search.context.FHIRSearchContext;
+import com.ibm.watson.health.fhir.search.util.SearchUtil;
+
+/**
+ * Bulk export Batchlet implementation.
+ * 
+ * @author Albert Wang
+ */
+public class BulkExportBatchLet implements Batchlet {
+    private AmazonS3 cosClient = null;
+
+    /**
+     * The IBM COS API key or S3 access key.
+     */
+    // eg "W00YiRnLW4a3fTjMB-oiB-2ySfTrFBIQQWanc--xxxxx"
+    @Inject
+    @BatchProperty(name = "cos.api.key")
+    String cosApiKeyProperty;
+
+    /**
+     * The IBM COS service instance id or s3 secret key.
+     */
+    // eg
+    // "crn:v1:bluemix:public:cloud-object-storage:global:a/3bf0d9003abfb5d29761c3e97696b71c:d6f04d83-xxxx-xxxx-xxxx-xxxxxxxxxxxx::"
```

Comment:
remove the 3bf0d9003abfb5d29761c3e97696b71c

Reply:
let me remove the whole sample